src/main/java/carrace/domain/CarName.java

```diff
@@ -0,0 +1,23 @@
+package carrace.domain;
+
+public class CarName {
+
+    private static final int LIMIT_NAME_LENGTH = 5;
+    private final String name;
+
+    public CarName(String name) {
+        validName(name);
+        this.name = name;
+    }
+
+    private void validName(String name) {
+        if (name.length() > LIMIT_NAME_LENGTH){
+            throw new IllegalArgumentException("자동차 이름은 5자를 초과할 수 없습니다.");
```

Comment:
```suggestion
            throw new IllegalArgumentException("자동차 이름은 " + LIMIT_NAME_LENGTH + "자를 초과할 수 없습니다.");
```
자동차 이름 길이 제한이 바뀌었을 때를 대비하는걸 추천드립니다

Reply:
오 이 꼼꼼함이라니... 저도 꼼꼼하다고 생각하는데 역시 세상은 넓네요 👍